Userland/Libraries/LibELF/DynamicLoader.cpp

```diff
@@ -482,7 +482,8 @@ DynamicLoader::RelocationResult DynamicLoader::do_relocation(const ELF::DynamicO
         // Apparently most loaders will just skip these?
         // Seems if the 'link editor' generates one something is funky with your code
         break;
-    case R_X86_64_64: {
+    case R_X86_64_64:
+    case R_AARCH64_ABS64: {
```

Comment:
very nit: I'd put the AARCH64 case label first, so that the case labels are alphabetically ordered. (also below.)

Reply:
Fixed!